routes/dbapi.js

```diff
@@ -15,32 +15,33 @@ function db() {
 }
 
 db.prototype.insertInCollection = function(collection, payload, callback) {
-	// validate(req.body)
-	let collec = _db.collection(collection);
-	// Check if word already exists
-	collec.find({word: payload.word}).toArray((err, results) => {
-		if (err) return callback(this.handleInsertionError(err));
-		if (results.length) return callback('Duplicate');
-
-		autoIncrement.getNextSequence(_db, collection, (err, autoIndex) => {
-			// Get auto incremented Id
-			payload._id = autoIndex;
-			collec.insert(payload, (err, result) => {
-				if (err) return callback(this.handleInsertionError(err));
-        return callback('Saved');
-			});
-		});
-	});
+  let collec = _db.collection(collection);
+  autoIncrement.getNextSequence(_db, collection, (err, autoIndex) => {
+    // Get auto incremented Id
+    payload._id = autoIndex;
+    collec.insert(payload, (err, result) => {
+      if (err) return callback(this.handleInsertionError(err));
+      return callback('Saved');
+    });
+  });
 };
 
+db.prototype.checkWordExists = function(collection, words, callback) {
+  let collec = _db.collection(collection);
+  collec.find({word: words}).toArray((err, results) => {
+    if (err) return callback(this.handleInsertionError(err));
+    callback(results.length ? 'Duplicate' : false);
+  });
+}
+
 db.prototype.retrieveGamesFollowingId = function(collection, id, callback) {
```

Comment:
do we still need this? i guess we can remove this, since now we are moving to random stuff